node/node_test.go

```diff
@@ -1360,3 +1362,99 @@ func TestNode_GetAccountAccountExistsShouldReturn(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, accnt, recovAccnt)
 }
+
+func TestNode_AppStatusHandlerWithPrometheusIncrementAndDecrementMetric(t *testing.T) {
+	t.Parallel()
+
+	metricKey := core.MetricIsSyncing
+	prometheus := statusHandler.NewPrometheusStatusHandler()
+	ashMock := mock.AppStatusHandlerStub{
+		IncrementHandler: func(key string) {
+			prometheus.Increment(key)
+		},
+		DecrementHandler: func(key string) {
+			prometheus.Decrement(key)
+		},
+		GetValueHandler: func(key string) float64 {
+			return prometheus.GetValue(key)
+		},
+	}
+
+	n, _ := node.NewNode(
+		node.WithAppStatusHandler(&ashMock))
+
+	ash := n.GetAppStatusHandler()
+
+	ash.Increment(metricKey)
+	ash.Increment(metricKey)
+	metricValue := ash.GetValue(metricKey)
+	assert.Equal(t, float64(2), metricValue)
+
+	ash.Decrement(metricKey)
+	metricValue = ash.GetValue(metricKey)
+	assert.Equal(t, float64(1), metricValue)
+}
+
+func TestNode_AppStatusHandlerWithPrometheusSetInt64AndUInt64ValueMetric(t *testing.T) {
+	t.Parallel()
+
+	metricKey := core.MetricNonce
+	//core.AppStatusHandler()
+	prometheus := statusHandler.NewPrometheusStatusHandler()
```

Comment:
Also here, I think is is useless to have these kind of tests

Reply:
fixed